README.md

```diff
@@ -0,0 +1,16 @@
+# UnderDog Dev's:  
```

Comment:
Don't believe there's an apostrophe in Devs because there wouldn't be one in Developers

Reply:
fixed

Reply:
ready to be merged